modules/local/run_esmfold.nf

```diff
@@ -0,0 +1,55 @@
+process RUN_ESMFOLD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://quay.io/nf-core/proteinfold_esmfold:1.1.0' :
+        'nf-core/proteinfold_esmfold:1.1.0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path ('./checkpoints/')
```

Comment:
Does it place all the files inside the `checkpoints` directory? thought that the way to declare it was path `('checkpoints/*')` or `('./checkpoints/*')` this is more to me to know that a change suggestion 😄

Reply:
Yes, it does. I didn't know either and tried just out of curiosity and it works